ui/packages/catalog-ui-search/src/main/webapp/component/input/boolean/input-boolean.view.js

```diff
@@ -13,12 +13,21 @@
  *
  **/
 
+import InputBoolean from './input-boolean.presentation'
+import * as React from 'react'
+
 var Marionette = require('marionette')
 var template = require('./input-boolean.hbs')
 var InputView = require('../input.view')
 
 module.exports = InputView.extend({
-  template: template,
+  template(props) {
```

Comment:
❗️ This is going to affect the way that filters looks when selecting a boolean attribute

Reply:
Reverted the change, should be fixed now